src/app/modules/registration/components/edit-images/edit-images.component.ts

```diff
@@ -102,6 +116,19 @@ export class EditImagesComponent implements OnInit {
   readonly onBeforeAdd = input<() => Promise<void> | void>();
   readonly attachmentType = input<AttachmentType>('Attachment');
   readonly ref = input<string>();
+  userSettings = toSignal(this.userSettingService.userSetting$);
+  selectedAspect = model<number | undefined>(undefined);
+
+  aspectOptions: SelectOption[] = [
```

Comment:
Jeg tror du vil finne nøyaktig samme kode andre steder i appen, se f.eks. for snøprofil:
![image](https://github.com/user-attachments/assets/6ab20bbc-bd8f-4194-800f-999e9d9b3c20)
Jeg tror du kan bruke "himmelretning" som ledetekst begge steder. Synes ikke "eksposisjon" er noe bra begrep på norsk.

Reply:
der brukes andre id-er. jeg må sende riktig id for himmelretning ikke tall fra 0-9. Sånn var det i den gamle løsningen så jeg tipper det er viktig at id er som en faktisk vindretning i grader (id)?

Reply:
Stemmer, det tenkte jeg ikke på. Da burde vi kanskje hatt en felles komponent som kunne gi oss både ID og grader, kanskje.